arch/powerpc/kernel/pci_dn.c

```diff
@@ -159,10 +159,8 @@ static struct pci_dn *add_one_dev_pci_data(struct pci_dn *parent,
 	pdn->parent = parent;
 	pdn->busno = busno;
 	pdn->devfn = devfn;
-#ifdef CONFIG_PPC_POWERNV
 	pdn->vf_index = vf_index;
 	pdn->pe_number = IODA_INVALID_PE;
```

Comment:
I dont know if you care about vf_index. It looks like on add_dev_pci_data call which calls this add_one_dev_pci_data function it will use the vf_index from a loop looking at pci_sriov_get_totalvfs. Your total vf's might not be the same as that I think

Reply:
Nvm I take this back you do care about vf_index you need it later. So then I guess if it just took the total VF's supported on the PF device and we use that as the total for numbering our vf_index's that is fine. This is probably good.

Reply:
At the moment I don't have a real argument for keeping it, as rtas calls are not defined in firmware. Might be helpful later for indexing relative to its pf. Conversely, I don't see the disadvantage for keeping it.

Reply:
I forgot that Jesse wanted the vf_index... Remember he said he needed it as part of that rtas call?